src/logistration/RegistrationPage.jsx

```diff
@@ -579,7 +579,7 @@ class RegistrationPage extends React.Component {
                 { this.state.enableOptionalField ? this.addExtraOptionalFields() : null}
                 <StatefulButton
                   type="button"
-                  className="btn-primary submit mt-4"
+                  className="btn-primary mt-4"
```

Comment:
Remove the `mt-4` here because the element above it has `margin-bottom` set on it.